docs/_includes/attr-use.adoc

```diff
@@ -4,24 +4,22 @@ Included in:
 - user-manual: Using attributes: set, assign, and reference
 ////
 
-Before you can use an attribute in your document, you must *set* it.
-Sometimes, setting an attribute is also referred to as *toggling on*.
-To set an attribute, you must add its name to the document.
+Before you can use an attribute in your document, it must be *set*.
+(Sometimes referred to as *toggling on*.)
 
-Once set, many attributes can be assigned a value.
-Values are the substitute text, content type, or operation that will replace any references to the attribute that the processor finds in the text.
+Some attributes are set for you when {adr} processes a document.
+Otherwise, you set an attribute by adding its name to the document:
```

Comment:
I think the phrase "adding its name to the document" is too vague. We should take this opportunity to make the meaning clearer. How about:

```
You can set (or override) an attribute on the document by declaring it in an attribute entry:
```